tests/tests/test-subscription.ts

```diff
@@ -493,6 +494,29 @@ describeWithMoonbeam(
 
       expect(data).to.not.be.null;
     });
+
+    step("should not receive log when contract fails", async function () {
+      const subscription = web3Subscribe("logs", {});
+
+      await new Promise((resolve) => {
+        subscription.once("connected", resolve);
+      });
+
+      await sendTransaction(context, {
+        gas: "0xD99BF", // lower than what is needed by 1
```

Comment:
Good comment. If we do make the needed value a constant, we should use that constant here.